main.go

```diff
@@ -120,7 +122,25 @@ func tagFileResources(path string, dir string, filter string, tags string, tfVer
 			} else {
 				log.Print("[INFO] Resource not taggable, skipping.", resource.Labels())
 			}
+		case "locals":
+			// Checks if terratag_added_* exists.
+			// If it exists no need to append it again to Terratag file.
+			// Instead should override it.
```

Comment:
We already have code that overwrites the terratag `locals` block
Seems like this code merges existing `locals` blocks with the terratag one.

Is that necessary?

Reply:
I don't believe there's code that merges existing Terratag locals. Where did you see it? (This is the code that I'm writing at the moment).

Reply:
I misunderstood, summary [here](https://github.com/env0/terratag/pull/118/files#r854117741)